_data/developer.yml

```diff
@@ -621,6 +614,13 @@ websites:
       - totp
     doc: https://support.microsoft.com/en-us/help/12408/
 
+  - name: Vonage
+    url: https://developer.nexmo.com/
```

Comment:
Wouldn't it be better to use vonage.com or what's the reasoning here?

Reply:
Multiple platforms all converging and it's still a process being undertaken. Nexmo is still at developer.nexmo.com, but called Vonage. Vonage also have a product called Vonage Business, which also happens to have a developer API, hence the direct link to this 😄 

I wish I had a more succinct answer for ya. Chances are, in the next few months, this will demand another PR for a URL update.